apps/prairielearn/src/components/ChangeIdButton.html.ts

```diff
@@ -68,6 +68,7 @@ function ChangeIdForm({
           name="id"
           value="${currentValue}"
           pattern="[\\-A-Za-z0-9_\\/]+"
+          aria-label="${label}"
```

Comment:
This shouldn't be necessary since this input has an associated `<label>` element.

Reply:
Removed in [b2ddc80](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/b2ddc80c9af2e30a4fa6ea94027150ce2ac8b3ba).